packages/@lwc/synthetic-shadow/src/env/node.ts

```diff
@@ -22,9 +22,26 @@ const {
     insertBefore,
     removeChild,
     replaceChild,
+    hasChildNodes,
+    contains,
 } = Node.prototype;
 
-const parentNodeGetter: (this: Node) => Element | null = getOwnPropertyDescriptor(
+const firstChildGetter: (this: Node) => ChildNode | null = getOwnPropertyDescriptor(
+    Node.prototype,
+    'firstChild'
+)!.get!;
+
+const lastChildGetter: (this: Node) => ChildNode | null = getOwnPropertyDescriptor(
+    Node.prototype,
+    'lastChild'
+)!.get!;
+
+const textContentGetter: (this: Node) => string = getOwnPropertyDescriptor(
+    Node.prototype,
+    'textContent'
+)!.get!;
+
+const parentNodeGetter: (this: Node) => (Node & ParentNode) | null = getOwnPropertyDescriptor(
```

Comment:
`(Node & ParentNode)` looks redundant, is the `Node` bit necessary?

Reply:
that's to match TS definitions so it doesn't complain about defining dom APIs with less general types.